doc/adding_lints.md

```diff
@@ -11,6 +11,7 @@ because that's clearly a non-descriptive name.
   - [Setup](#setup)
   - [Getting Started](#getting-started)
   - [Testing](#testing)
+    - [Cargo lints](#cargo-lints)
```

Comment:
Ninja fix :smile:

Reply:
xD even for me. A IDE plugin did that and I only noticed it when I reviewed the changes on GitHub xD

I would also suggest merging this last, another idea I had was to crate a rollup PR that combines all metadata related PRs :thinking: You'll find a good solution, I have faith in you :upside_down_face: